packages/fast-components-react-msft/src/hypertext/examples.data.ts

```diff
@@ -0,0 +1,16 @@
+import {ICategoryItemProps} from "@microsoft/fast-development-site-react";
+import {IGenericExample} from "../examples";
```

Comment:
I think you'll want to do a rebase and replace IGenericExample with ISnapshots....  Little bit of a refactor, unfortunately.